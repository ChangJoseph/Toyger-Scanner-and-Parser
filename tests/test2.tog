//a test case with variables
let  //start
  var x:=1  //variable with init value
in  // done with declarations
  printint(x)  //statements
end //last comment!
