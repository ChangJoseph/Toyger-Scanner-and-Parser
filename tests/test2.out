Global Variable x: line 3

Number of functions defined: 0
Number of comments: 6
Number of strings: 0
